cmd/eksctl-anywhere/cmd/listpackages.go

```diff
@@ -0,0 +1,61 @@
+package cmd
+
+import (
+	"context"
+	"log"
+
+	"github.com/spf13/cobra"
+
+	"github.com/aws/eks-anywhere/pkg/curatedpackages"
+	"github.com/aws/eks-anywhere/pkg/kubeconfig"
+)
+
+type listPackagesOption struct {
+	kubeVersion string
+	source      string
+}
+
+var lpo = &listPackagesOption{}
+
+func init() {
+	listCmd.AddCommand(listPackagesCommand)
+	listPackagesCommand.Flags().StringVar(&lpo.source, "source", "", "Discovery Location. Options (cluster, registry)")
+	listPackagesCommand.Flags().StringVar(&lpo.kubeVersion, "kubeversion", "", "Kubernetes Version of the cluster to be used. Format <major>.<minor>")
+	err := listPackagesCommand.MarkFlagRequired("source")
+	if err != nil {
+		log.Fatalf("Error marking flag as required: %v", err)
+	}
+}
+
+var listPackagesCommand = &cobra.Command{
+	Use:          "packages",
+	Short:        "Generate a list of curated packages available to install",
+	Long:         "This command is used to generate a list of curated packages available to install",
```

Comment:
```suggestion
	Long:         "This command is used to generate a list of curated packages available to install",
```
Probably no need for this, since it doesn't add anything over the short value.